### DISCUSSION TITLE
After upgrading to the newest Ubuntu 22.04.01 LTS can't install chia

### DISCUSSION THREAD

**Discussion by randomisresistance** [Chia-Network/chia-blockchain#13188] [Support]
Can't install or start the chia client after upgrading to newest LTS release of Ubuntu.

```
Installing on Ubuntu 22.* or newer.
[sudo] password for chia: 
Hit:1 https://mirrors.opensource.is/ubuntu jammy InRelease
Hit:2 https://mirrors.opensource.is/ubuntu jammy-updates InRelease
Hit:3 https://mirrors.opensource.is/ubuntu jammy-backports InRelease
Hit:4 https://mirrors.opensource.is/ubuntu jammy-security InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
openssl is already the newest version (3.0.2-0ubuntu1.6).
python3.10-venv is already the newest version (3.10.4-3ubuntu0.1).
The following package was automatically installed and is no longer required:
  libffi7
Use 'sudo apt autoremove' to remove it.
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Python version is 3.10
SQLite version for Python is 3.37.2
OpenSSL version for Python is OpenSSL 3.0.2 15 Mar 2022
install.sh: 293: python: not found
chia@harvester:~/chia-blockchain$ . ./activate
(venv) chia@NUC10i7FNH:~/chia-blockchain$ chia version
-bash: /home/chia/chia-blockchain/venv/bin/chia: /home/chia/chia-blockchain/venv/bin/python: bad interpreter: No such file or directory


```

**Comment by randomisresistance** [comment]
https://github.com/Chia-Network/chia-blockchain/issues/11113

Gives a good manual how to fix this issue, but since 22.04.01 LTS is the newest LTS release and is out now for about 4 month I think this issue should be fixed. Or is there a big showstopper in python 3.10.* ?  

RiR

**Comment by MartijnVanAndel** [comment]
I had the same error trying to upgrade from source code according to: https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#update-from-source 

In my situation your stated problem was caused by the fact that Ubuntu 22.04.1 uses `python3` instead of `python` as command. 
After changing this I got troubles with my _venv_ environment in the chia-blockchain folder. 

I solved this issue in three steps:

1. after `git status` shows you have reset to the latest git branch, change the file `install.sh`, row 293 untul 298 to:  
```
python3 -m pip install --upgrade pip
python3 -m pip install wheel
#if [ "$INSTALL_PYTHON_VERSION" = "3.8" ]; then
# This remains in case there is a diversion of binary wheels
python3 -m pip install --extra-index-url https://pypi.chia.net/simple/ miniupnpc==2.2.2
python3 -m pip install -e ."${EXTRAS}" --extra-index-url https://pypi.chia.net/simple/
```
2. Then remove the `venv` folder in your `chia-blockchain` base folder (in my case: ~/repos/chia-blockchain)
3. run `sh install.sh` and continue the instructions

**Reply by MartijnVanAndel** [reply]
So this instruction let's chia-blockchain run on the default python 3.10 version of 22.04, without the need to install python 3.9. Other how to's, including your link, seemed to instruct how to install python 3.9 and use that second environment.

**Reply by randomisresistance** [reply]
Thanks for your reply!
After reading install.sh I figured out that this should work already. But you have to `rm  venv` folder this also fixes the issue. Even after Dist-Upgrade from an older Ubuntu version.
